app/src/main/java/com/android/unio/model/event/Event.kt

```diff
@@ -53,12 +53,18 @@ data class Event(
     val endDate: Timestamp = Timestamp(Date()),
     val location: Location = Location(),
     val types: List<EventType> = mutableListOf(EventType.OTHER),
-    val placesRemaining: Int = -1,
+    val maxNumberOfPlaces: Int = -1,
```

Comment:
What's this name change for? Anyways I think we don't even use this as intended. But I would agree, if that's what you thought when making this change, that it would be better to simply store the max number of places, because we can easily compute the number of places remaining from it. This prevents us from making update queries for this field every time a user signs up to the event.

Reply:
Yes precisely, I found it better to have a static max number of places and compute the remaining places as maxNumberOfPlaces - numberOfSaved